doc/api/status.rst

```diff
@@ -22,6 +22,10 @@ Or, using the more verbose name:
 Using these constants helps avoid typos and cuts down on the number of
 string objects that must be created when preparing responses.
 
+Falcon also provides a generic 'HTTPStatus' class. Raise this class from a hook,
```

Comment:
Right, although in this case we don't have any predefined child classes, so saying "Base Class" may be confusing. You  could simply use "HTTPStatus" as the heading instead.